### PR TITLE
docs(controller-config): fix wording on agent controller config keys

### PR DESCRIPTION
**Pull request by nvinuesa**
This patch improves the wording on these four controller config keys:
- agent-ratelimit-max
- agent-ratelimit-rate
- agent-logfile-max-size
- agent-logfile-max-backups

## Checklist

- [ ] Code style: imports ordered, good names, simple structure, etc
- [ ] Comments saying why design decisions were made
- [ ] Go unit tests, with comments saying what you're testing
- [ ] [Integration tests](https://github.com/juju/juju/tree/main/tests), with comments saying what you're testing
- [ ] [doc.go](https://discourse.charmhub.io/t/readme-in-packages/451) added or updated in changed packages

## Links

**Launchpad bug:** https://bugs.launchpad.net/juju/+bug/

**Jira card:** JUJU-

### CODE REVIEW

File: controller/config.go [hunk 1]
```diff
@@ -165,12 +165,12 @@ const (
 	// which should be more than enough time for a debugging session.
 	MaxDebugLogDuration = "max-debug-log-duration"
 
-	// AgentLogfileMaxSize is the maximum file size in MB of each
-	// agent/controller log file.
+	// AgentLogfileMaxSize is the maximum file size in MB of each agent log
```
Comment (tmihoc) [inline controller/config.go:168]: For consistency with the previous example, I'd move the unit to the end:

> AgentLogfileMaxSize is the maximum file size of each agent log, in MB.

File: controller/config.go [hunk 2]
```diff
@@ -165,12 +165,12 @@ const (
...
+	// file.
 	AgentLogfileMaxSize = "agent-logfile-max-size"
 
-	// AgentLogfileMaxBackups is the number of old agent/controller log files
-	// to keep (compressed).
+	// AgentLogfileMaxBackups is the number of old agent log files to keep
+	// (compressed; saved on each unit, synced to the controller).
 	AgentLogfileMaxBackups = "agent-logfile-max-backups"
```
Comment (tmihoc) [inline controller/config.go:174]: Add "maximum":

```
// AgentLogfileMaxBackups is the maximum number of old agent log files to keep
// (compressed; saved on each unit, synced to the controller).
```

**Review by tmihoc — CHANGES_REQUESTED** [review]
A couple of small suggestions. PS Out of scope for this PR, but any chance we could record config units (MB, ms, etc.) in a separate variable?

File: controller/config.go (hunk 2 above)
Comment (nvinuesa) [reply controller/config.go:174]: Fixed.

File: controller/config.go (hunk 1 above)
Comment (nvinuesa) [reply controller/config.go]: Fixed.

**Review by tmihoc — APPROVED** [review]
LGTM, thanks!

**Pull request merged**